talk/expert/perfectforwarding.tex

```diff
@@ -57,7 +57,6 @@
   \begin{alertblock}{}
     \begin{itemize}
       \item \mintinline{cpp}{const T&} won't work when passing something non const
-      \item rvalues are not supported in either case
```

Comment:
Why is this removed? It should be at least reworded in: `rvalues are also not supported`. In the example `wrapper(42)`, `42` is an rvalue (a prvalue), and cannot be bound to a mutable reference. This is important here, because otherwise `wrapper(T& arg)` could handle all cases.

Reply:
True. I would prefer to have that in the summary of the slide at the bottom though. But if you insist, we can leave it as is.